problems/strings.js

```diff
@@ -103,15 +191,70 @@ function disemvowel() {}
  * @returns {string}
  */
 
- function disemvowelLongWords(){}
+// stringYouWishToSplitIntAnArray.slice(characterYouWishToSplit)
+// Popular Splits -
+// str.split(“”) - separates every character .
+// str.split(“ “) - splits words
+
+const disemvowelLongWords = (string) => {
+  let newSentence = "";
+  let noVowels = "";
+  let newString = string.split(" ");
+  for (let i = 0; i < newString.length - 1; i++) {
+    word = newString[i]; // [cat bat lap] new string is the word
+    if (word.length >= 5) {
+      //remove vowels
+      for (let i = 0; i < word.length - 1; i++) {
```

Comment:
You're doing the same here where you won't be able to reach the last array element `newString.length - 1`